src/FillArrays.jl

```diff
@@ -572,6 +572,12 @@ count(x::Ones{Bool}) = length(x)
 count(x::Zeros{Bool}) = 0
 count(f, x::AbstractFill) = f(getindex_value(x)) ? length(x) : 0
 
+#########
+# in
+#########
+in(x, A::Union{Ones, Zeros, Fill}) = x == getindex_value(A)
```

Comment:
why not `A::AbstractFill`?

Reply:
Fair enough, given that these are the only subtypes of `AbstractFill`